rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java

```diff
@@ -93,6 +104,15 @@ public void setAllCached(String remotePath, long fileLength, long startBlock, lo
         }
     }
 
+    @Override
+    public Map getCacheStats()
+    {
+        Map<String, Double> stats = new HashMap<String, Double>();
+        stats.put("Cache Hit Rate",  ((double)cachedRequests/totalRequests));
```

Comment:
leave space after (double) ... we need to add checkstyle too, will open issue for that